packages/handlers/openapi/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@graphql-mesh/openapi",
-  "version": "0.31.2",
+  "version": "0.32.0",
```

Comment:
I think it should be a major bump, right @ardatan?

Reply:
I know that mesh is not yet 1.0 but we should do a major bump on this package

Reply:
We are doing minor bumps with `BREAKING CHANGE` notice in the changelog.